File: 02774.py
def beauty(n):
    num = []
    while n > 0:
        if n % 10 not in num:
            num.append(n % 10)
        n //= 10

    return len(num)

for i in range(int(input())):
    X = int(input())
    print(beauty(X))

File: 02774.cpp
#include <iostream>
using namespace std;

int beauty(int n) {
	int num[10];
	for (int i = 0; i < 10; i++)
		num[i] = 0;

	while (n > 0) {
		num[n % 10]++;
		n /= 10;
	}

	int cnt = 0;
	for (int i = 0; i < 10; i++)
		if (num[i]) cnt++;

	return cnt;
}

int main() {
	int T; cin >> T;

	for (int i = 0; i < T; i++) {
		int X; cin >> X;
		cout << beauty(X) << endl;
	}
}
